src/base/string_buffer.hpp

```diff
@@ -91,7 +91,7 @@ class string_buffer: public mem::block_ext //collect formatted text (printf-like
 
     void add_bytes (size_t len, void *bytes);                     //add "len" bytes (can have '\0' in the middle)
 
-    template<typename... Args> inline void operator() (Args &&... args); //add with printf format
+    template<typename... Args> inline int operator() (Args &&... args); //add with printf format
```

Comment:
Is return value ever used? Please explain the change.

Reply:
the return value is not used yet but reviewing the code I found places where this value could be useful, eliminating the need of additional buffers; I put it there for future use

Reply:
Can you show me an example?

Reply:
db_value_printer.cpp: describe_real()
if we know how many chars have been added then we can revert it in place and add a different text

Reply:
ok